filters/openpolicyagent/common.go

```diff
@@ -0,0 +1,7 @@
+package openpolicyagent
+
+const (
+	OpenPolicyAgentDecisionKey        = "open-policy-agent:decision"
+	OpenPolicyAgentDecisionBodyKey    = "open-policy-agent:decision-body"
+	OpenPolicyAgentDecisionHeadersKey = "open-policy-agent:decision-headers"
```

Comment:
I would move all const to filters/openpolicyagent/openpolicyagent.go and drop this file